components/Accordion/src/index.scss

```diff
@@ -1,162 +1,6 @@
-  transform: var(--denhaag-accordion-icon-transform);
-}
-
-.denhaag-accordion__title {
-  font-family: inherit;
-  font-weight: inherit;
-  font-size: inherit;
-  background: inherit;
-  border: inherit;
-  text-align: inherit;
-  hyphens: auto;
-  margin-block-start: 0;
-  margin-block-end: 0;
-  flex-grow: var(--denhaag-accordion-title-flex-grow);
-  padding-block-start: var(--denhaag-accordion-title-padding-block);
-  padding-block-end: var(--denhaag-accordion-title-padding-block);
-  padding-inline-start: var(--denhaag-accordion-title-padding-inline);
-  padding-inline-end: calc(
-    (var(--denhaag-accordion-title-padding-inline) * 2) + var(--denhaag-accordion-icon-width, 0)
-  );
-}
-
-.denhaag-accordion__title--focus,
-.denhaag-accordion__title--focus + .denhaag-icon,
-.denhaag-accordion__panel--hover .denhaag-accordion__title,
-.denhaag-accordion__panel--hover .denhaag-icon,
-.denhaag-accordion__title:focus-visible {
-  color: var(--denhaag-accordion-panel-color);
-  outline: var(--denhaag-accordion-panel-outline);
-}
-
-.denhaag-accordion__container--open .denhaag-accordion__title,
-.denhaag-accordion__container--open .denhaag-icon {
-  color: inherit;
-}
-
-.denhaag-accordion__title--disabled,
-.denhaag-accordion__title[disabled],
-.denhaag-accordion__title--disabled + .denhaag-icon,
-.denhaag-accordion__title[disabled] + .denhaag-icon {
-  z-index: 1;
-}
-
-.denhaag-accordion__title:focus-visible,
-.denhaag-accordion__title:focus-visible + .denhaag-icon {
-  color: var(--denhaag-accordion-panel-color);
-  outline: var(--denhaag-accordion-panel-outline);
-}
-
-.denhaag-accordion__container--open .denhaag-accordion__panel--hover .denhaag-accordion__title,
-.denhaag-accordion__container--open .denhaag-accordion__panel--hover .denhaag-icon {
-  color: var(--denhaag-accordion-container-open-color);
-}
-
-.denhaag-accordion__container--open .denhaag-accordion__panel:hover .denhaag-accordion__title,
-.denhaag-accordion__container--open .denhaag-accordion__panel:hover .denhaag-icon {
-  color: var(--denhaag-accordion-container-open-color);
-}
-
-.denhaag-accordion__panel:hover .denhaag-accordion__title:not([disabled]),
-.denhaag-accordion__panel:hover .denhaag-accordion__title:not([disabled]) + .denhaag-icon {
-  color: var(--denhaag-accordion-panel-color);
-  outline: var(--denhaag-accordion-panel-outline);
-}
-
-.denhaag-accordion__title--disabled::before,
-.denhaag-accordion__title[disabled]::before {
-  content: "";
-  width: 100%;
-  height: 100%;
-  position: absolute;
-  top: 0;
-  left: 0;
-  background-color: var(--denhaag-accordion-title-disabled-background-color);
-  z-index: -1;
-}
-
-.denhaag-accordion__title--focus::after,
-.denhaag-accordion__title:focus-visible::after {
-  border-radius: var(--denhaag-accordion-title-focus-border-radius);
-  content: "";
-  height: 100%;
-  left: 0;
-  outline: var(--denhaag-accordion-title-focus-outline);
-  position: absolute;
-  top: 0;
-  width: 100%;
-}
```

Comment:
De focus state is bij mij nu gone / browser default. Ik denk dat dat niet de bedoeling was?

<img width="854" alt="image" src="https://github.com/user-attachments/assets/4d3f84b7-fb20-4dfb-98e5-45db7fed86ca" />

Reply:
Nee klopt, heb nu de default focus state er op gezet.